tools/src/shellcheck.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+SHELLCHECK_VERSION=v0.8.0
+GOARCH=$(go env GOARCH)
+GOOS=$(go env GOOS)
+
+cd tools/bin && wget -qO- "https://github.com/koalaman/shellcheck/releases/download/${SHELLCHECK_VERSION}/shellcheck-${SHELLCHECK_VERSION}.${GOOS}.x86_64.tar.xz" | tar -xJv 
```

Comment:
do not `wget` every time run this script?

Reply:
Thanks for raising this. Yes, we should check if binary existed.